generic_config_updater/gcu_table_modification_validators.conf.json

```diff
@@ -0,0 +1,39 @@
+{
+    "README": [
+        "table_modification_validators provides, module & method name as ",
+        "    <module name>.<method name>",
+        "NOTE: module name could have '.'",
+        " ",
+        "The last element separated by  '.' is considered as ",
+        "method name",
+        "",
+        "e.g. 'show.acl.test_acl'",
+        "",
+	"table_modification_validators_and for a given table defines a list of validators that all must pass for modification to the table to be allowed",
+	"table_modification_validators_or for a given table defines a list of validators, at least one of which must pass for modification of the table to be allowed",
+        ""
+    ],
+    "tables": {
+	"PFC_WD": {
```

Comment:
could you reformat? mixing tabs and spaces now.

Reply:
Changed to all spaces